dockstore-webservice/src/main/resources/swagger.yaml

```diff
@@ -3310,6 +3310,46 @@ paths:
             type: "array"
             items:
               $ref: "#/definitions/Workflow"
+  /workflows/{entryId}/registerCheckerWorkflow/{descriptorType}:
+    post:
+      tags:
+      - "workflows"
+      summary: "Register a checker workflow and associates it with the given tool/workflow"
+      description: ""
+      operationId: "registerCheckerWorkflow"
+      produces:
+      - "application/json"
+      parameters:
+      - name: "checkerWorkflowPath"
+        in: "query"
+        description: "Path of the main descriptor of the checker workflow (located\
+          \ in associated tool/workflow repository)"
+        required: true
+        type: "string"
+      - name: "testParameterPath"
+        in: "query"
+        description: "Default path to test parameter files for the checker workflow.\
+          \ If not specified will use that of the entry."
+        required: false
+        type: "string"
+      - name: "entryId"
+        in: "path"
+        description: "Entry Id."
+        required: true
+        type: "integer"
+        format: "int64"
+      - name: "descriptorType"
+        in: "path"
+        description: "Descriptor type."
+        required: true
+        type: "string"
```

Comment:
Possibly change to enum (like some of the GA4GH endpoints) or put some examples in the description (path parameter `example` properties don't work in swagger 2.0)